template/default.tmpl

```diff
@@ -1,5 +1,5 @@
 {{ define "__alertmanager" }}AlertManager{{ end }}
-{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts{{ end }}
+{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}{{ end }}
```

Comment:
Sorry for my confusion, but the initial example in the PR's description says it all, actually:

```
curl -sS 'http://localhost:9093/api/v1/alerts?silenced=false&receiver=webhook\d' | jq .data[].annotations.receiver
"webhook1"
"webhook1"
"webhook2"
"webhook2"
"webhook1"
"webhook1"
"webhook2"
"webhook2"
```

Thus, the backend does the filtering with a regexp (which is good), so at some point between the receiver as specified in `alertmanager.yml` and the receiver as added as a URL parameter in the template, we need to do the regexp escaping.